encryption/client_side_encryption.py

```diff
@@ -0,0 +1,39 @@
+import hashlib
+import aws_encryption_sdk
+from aws_encryption_sdk.identifiers import EncryptionKeyType, WrappingAlgorithm, CommitmentPolicy
+from aws_encryption_sdk.key_providers.raw import RawMasterKeyProvider
+from aws_encryption_sdk.internal.crypto.wrapping_keys import WrappingKey
+
+class ClientSideEncryption():
```

Comment:
I'm not sure this is worth a separate file (mostly because it is going to make packaging a bit of a pain).